Library/Formula/tarantool.rb

```diff
@@ -0,0 +1,60 @@
+class Tarantool < Formula
+  def instances_enabled
+    etc+"tarantool/instances.enabled"
+  end
+
+  desc "In-memory database and Lua application server."
+  homepage "http://tarantool.org"
+  url "http://tarantool.org/dist/master/tarantool-1.6.7-562-g3dbc9ff-src.tar.gz"
+  version "1.6.7-562"
+  sha256 "6aded257e11fa114dc21e8493ff96dfe3f8a692d62e096afb5105cecc179211d"
+
+  devel do
+    url "https://github.com/tarantool/tarantool.git", :branch => "1.6", :shallow => false
```

Comment:
Yes, by default git makes shallow copy and Tarantool needs git tags version detection.